omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java

```diff
@@ -231,10 +231,10 @@ public PatientIdentifier save(PatientIdentifier delegate) {
 		}
 		
 		if (needToAdd) {
+			service().savePatientIdentifier(delegate);
 			delegate.getPatient().addIdentifier(delegate);
 		}
-		
-		service().savePatientIdentifier(delegate);
+
```

Comment:
This seems unrelated to this ticket, was this done to support a different issue?

Reply:
I don't know how the unrelated commits are tagged up on this issue branch.